libraries/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -137,6 +171,7 @@ bool _IotMqtt_ValidateConnect( const IotMqttConnectInfo_t * pConnectInfo )
                          "the maximum supported length of %hu.",
                          pConnectInfo->clientIdentifierLength,
                          maxClientIdLength );
+            IOT_SET_AND_GOTO_CLEANUP( false );
```

Comment:
This was removed in PR #608, adding it back.

Reply:
Add newline before 175

Reply:
Done